Update web-sites-dotnet-store-data-mongodb-vm.md

jboeshart opened the pull request:
Added default value to the date the task is due since the default sets it to 1/1/0001, just makes it a little cleaner.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jboeshart committed:
Update web-sites-dotnet-store-data-mongodb-vm.md

Added default value to the date the task is due since the default sets it to 1/1/0001, just makes it a little cleaner.
